From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Cao jin <caoj.fnst@cn.fujitsu.com>
Date: Tue, 3 Jul 2018 18:51:13 +0800
Subject: [PATCH] grub-setup: Debug message cleanup

Variable "root" is initialized after root device probing and is null in
current place, so, drop it.

Signed-off-by: Cao jin <caoj.fnst@cn.fujitsu.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 util/setup.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/util/setup.c b/util/setup.c
index 80363075d34..9c1e1b7da6a 100644
--- a/util/setup.c
+++ b/util/setup.c
@@ -305,9 +305,8 @@ SETUP (const char *dir,
   bl.first_block = (struct grub_boot_blocklist *) (core_img
 						   + GRUB_DISK_SECTOR_SIZE
 						   - sizeof (*bl.block));
-  grub_util_info ("root is `%s', dest is `%s'", root, dest);
 
-  grub_util_info ("Opening dest");
+  grub_util_info ("Opening dest `%s'", dest);
   dest_dev = grub_device_open (dest);
   if (! dest_dev)
     grub_util_error ("%s", grub_errmsg);
